buildspec.yml

```diff
@@ -0,0 +1,22 @@
+version: 0.2
+
+phases:
+  pre_build:
+    commands:
+      - mkdir -p /root/.gradle
+      - echo "org.gradle.daemon=false" > /root/.gradle/gradle.properties
+  build:
+    commands:
+      - ./gradlew build
```

Comment:
When I run the build from my command line, I usually run ./gradlew clean build to make sure it didn't show me wrong error messages. Should we also change this into clean build?

Reply:
We don't need the `clean`. The CodeBuild or Travis will always start a build from a fresh container.

Reply:
Good to know that! Then I think this is fine. 
It's better to let others check this PR as well, given that I'm not that familiar with CodeBuild.

Reply:
yep, Jacob has reviewed this PR.